packages/icons/package.json

```diff
@@ -3,5 +3,17 @@
   "version": "1.0.1-alpha.29",
   "description": "svg icons for Mozaic design system",
   "author": "",
-  "license": "ISC"
+  "main": "index.js",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.24.1",
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "babel-preset-env": "^1.5.1",
+    "rimraf": "^3.0.0"
```

Comment:
I don't think rimraf is used here, only at the root.
Maybe those dependencies should stay at the root, but are they used yet ?
if not, remove everything.

Reply:
Actually rimraf is used in iconCompiler dir, and It's not present in root package.json. Anyway, I'll remove it from here and install it in root, so It can be used anywhere. All the other dependencies are not necessary in this PR, but they will be in "part 2".